vscode-web/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no" />
+        <title>VSCode Web</title>
```

Comment:
Do we need to add our branding here, something like `VSCode Web - github1s` ?

Reply:
```suggestion
        <title>VSCode Web - github1s</title>
```

Reply:
This HTML is not used in GitHub1s, it is simply used to test whether @github1s/vscode-web (the web version of vscode) can run independently.

Reply:
ok, yes. Fine!